environment/qa4sm_env.yml

```diff
@@ -166,7 +166,7 @@ dependencies:
     - pytest-cov==2.10.1
     - pytest-django==4.1.0
     - pytest-mpl==0.12
-    - qa4sm-reader==0.3.3
+    - git+https://github.com/sheenaze/qa4sm-reader.git@update_irregular_plots
```

Comment:
Can you update to the original repo? I think it would be a good compromise to say that pip install from github is ok in master, but we should point to original qa4sm-reader at least (and for deployment/tag/release we have to use a released pypi version of the reader)

Reply:
and in your fork or any non-master branch we can point to any branch in any fork of external packages (like you did here)

Reply:
yes, of course, I just wanted to make sure that it works :)
give me a minute.